src/main/java/hudson/plugins/gradle/GradleInstallation.java

```diff
@@ -68,28 +73,46 @@ public void buildEnvVars(EnvVars env) {
         env.put("PATH+GRADLE", getHome() + "/bin");
     }
 
-    public String getExecutable(Launcher launcher) throws IOException, InterruptedException {
-        return launcher.getChannel().call(new MasterToSlaveCallable<String, IOException>() {
-            public String call() throws IOException {
-                File exe = getExeFile();
-                if (exe.exists()) {
-                    return exe.getPath();
-                }
-                return null;
+    public String getExecutable(final Launcher launcher) throws IOException, InterruptedException {
+        final File exe = launcher.getChannel().call(new MasterToSlaveCallable<File, IOException>() {
+
+            private static final long serialVersionUID = 5572701806697595613L;
```

Comment:
What is the `serialVersionUID` for? IIUC, Jenkins uses XStream to serialize/deserialize data, not Java serialization. is this required?

Reply:
Yes you are right, it's not needed here. Just a habit from my daily work. I'll remove it.